src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,85 @@
+package duke.task;
+
+import duke.util.Storage;
+
+import java.util.ArrayList;
+
+/**
+ * Class the holds the tasks provided by the user.
+ */
+public class TaskList {
```

Comment:
The class has a clean structure and the methods are following SOLID principles well.